frontend/src/v5/services/api/drawingRevisions.ts

```diff
@@ -27,15 +27,18 @@ const revisions: IDrawingRevision[] = [
 	{
 		_id: uuid(),
 		name: 'rev1',
-		timestamp: 1409009331628,
+		timestamp: new Date(1409009331628),
```

Comment:
There some inconsistencies with the mocked data.  It would be good to have everything consistent for a good demo. If it's too complicated lets  just fix the one in bold

- All drawings have the same revisions
- The revision count is inconsistent 
- **After uploading a revision the revision is not there**

<img width="1138" alt="image" src="https://github.com/3drepo/3drepo.io/assets/42032433/8d6df560-96f6-4b1e-9ba2-2a46a401a390">